### DISCUSSION TITLE
spline not capturing logs

### DISCUSSION THREAD

**Discussion by zacayd** [AbsaOSS/spline-getting-started#34] [Q&A]
Hi 
i installed the spline on the  azure machine  with the docker  in order to catch lineage on databricks on azure
```shell
docker-compose up
```

i got the page 
http://192.168.100.11:8080/index.html
I put on the  avdansed setting

```
spark.spline.lineageDispatcher.http.producer.url http://192.168.100.11:8080/producer
spark.spline.lineageDispatcher http
```

but cannot see lineage on  
http://192.168.100.11:9090/app/events/list

please advise
where can i find logs
Zacay
zacayd@octopai.com

**Comment by wajda** [comment]
What do you mean by that? What logs are you looking for? Spline has many components. If you run docker all logs are inside docker containers. Spline agent logs are where Spark logs are (it writes to Spark logs).

**Reply by zacayd** [reply]
How can i requst the rest-api of spline?

**Reply by wajda** [reply]
Please try to make your questions specific, clear, and post them into corresponding threads.
